tests/tck/features/index/Index.IntVid.feature

```diff
@@ -125,9 +125,9 @@ Feature: IndexTest_Vid_Int
       Show TAG INDEXES;
       """
     Then the result should be, in any order:
-      | Names                |
-      | "disorder_tag_index" |
-      | "single_tag_index"   |
+      | Index Name           | By Tag  | Columns      |
+      | "disorder_tag_index" | "tag_1" | "col3, col2" |
```

Comment:
I think you could return columns as list type.

Reply:
agree

Reply:
> agree, it can help users to get column name

For example?

Reply:
user can use it to check if the index columns are expected, the list is easier to handle than string.

Reply:
> user can use it to check if the index columns are expected, the list is easier to handle than string.

Is it just an output format opinion? I think that's enough for the moment, it should be the same as `SHOW CREATE ...`.
In addition, if the user needs to determine whether an index is valid or not, this needs to be combined with the execution plan.